src/main/java/com/example/enterpriseapplication/dto/Recipe.java

```diff
@@ -12,4 +13,14 @@ class Recipe {
     private String recipeName;
     private String[] recipeSteps;
     private String[] recipeIngredients;
+
+    @Override
+    public String toString() {
+        return "Recipe{" +
+                "recipeID=" + recipeID +
+                ", recipeName='" + recipeName + '\'' +
+                ", recipeSteps=" + Arrays.toString(recipeSteps) +
+                ", recipeIngredients=" + Arrays.toString(recipeIngredients) +
+                '}';
+    }
```

Comment:
Added a toString override since in a duplicate Recipe class the group added that functionality.